pom.xml

```diff
@@ -326,6 +326,17 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-site-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.maven.surefire</groupId>
+                        <artifactId>surefire-junit47</artifactId>
```

Comment:
Is there a specific reason why surefire-junit47 is used there instead of regular junit?

Reply:
There is an older runner in classpath, it is picked automatically but it will not find any test. Removing it is a bigger effort, which is out of scope of this upgrade.